database/migrations/2018_09_06_150943_create_bundles_table.php

```diff
@@ -16,16 +16,21 @@ public function up()
         Schema::create('bundles', function (Blueprint $table) {
             $table->increments('id');
             $table->integer('entitlement')->unsigned(); // for recording actual entitlement when issued
-            $table->integer('family_id')->unsigned(); // FK Family
-            $table->integer('centre_id')->unsigned(); // FK Centres
+            $table->integer('registration_id')->unsigned(); // FK Registrations
+            $table->integer('allocating_centre_id')->unsigned(); // FK Centres
```

Comment:
Annoying so they can be different centres?

Reply:
There can now be an allocating centre (where they made the bundle) and a disbursing centre (where they *shipped* the bundle to be given out). Fits a behaviour in a couple of places.